docu/files/Multi-tabEditor.html.md

```diff
@@ -0,0 +1,96 @@
+    $scope.localizedData = {
+        'en': 'english text',
+        'de': 'german text',
+        'pl': 'polish text'};
+```
+Remember that a list of languages (currencies) in which the property is localized has to match the list of defined languages (currencies) for the current user.
+* HTML markup
+```html
+    <div class="panel-body">
+        <h3>Single Line</h3>
+        <builder-tab-editor builder-locales="configuration" data="localizedData" ng-form="someName2" ></builder-tab-editor>
+    </div>
+```
+
+#### Attributes: 
+
+- <b>data</b> - Required. Indicates which text should fill the editor prior to user input. 
+- <b>builder-validate-tabs</b> - Not required. Default false. Specifies whether a field should be validated or is required in order to save.
+- <b>builder-locales</b> - A structure specifing which locale is default and/or must be filled. This attribute enables you to choose between a multi-tab editor for languages and for currencies. You must set the value to `languages` or `currencies`, respectively. That attribute is required for a localized editor component. Please avoid configuration that will render your code invalid. For example, a multi-line currency editor is an example of configuration that is not supported.
+The <b>builder-editor-type</b> attribute defines the type of editor. If it is empty, that indicates a single-line editor. If it contains a tab, that indicates a multi-line editor with tabs for languages. For the multi-line editor, you can specify the number of rows using the <b>builder-editor-rows</b> attribute. The <b>localizedData</b> object is passed on the <b>data</b> attribute, and the localized field label is passed with the <b>label</b> attribute.
+
+### Validate the multi-tab editor
+
+f you want your editor to be validated, you need to add the builder-validate-tabs attribute set as true to the definition of the editor. Every value is validated.
+
+There are two warning indicators used in the validation process:
+
+<ul>
+<li> **globe** - Informs indirectly that one or more fields with required language from the list is empty.</li>
+<li> **frame text box** - Informs directly which field from the required ones you need to fill in.</li>
+</ul>
+The validation process consists of two stages. You can find more detailed information about each of them below:
+
+<ol>
+<li> On the load view, warning indicators are highlighted in orange if one or more required fields are empty. This stage is implemented out of the box if you apply localized editor validation.</li>
+<br>
+<li> Optionally, you can broadcast the following inside your controller to add one more validation stage:
+<br><br>
+```js
+    $rootScope.$broadcast($scope.someName2.$name + '-validate-on-save');
+```
+<br><br> In this second stage, when user tries to save or publish the item and there are required text boxes that are still empty or filled incorrectly, your localized editor is re-validated. The event is broadcast and both indicators turn red.</li>
```

Comment:
Add article, remove hyphen, passive voice (please revise if my changes don't make sense), and extra text: 
In this second stage, when a user tries to save or publish the item and required text boxes are empty or filled incorrectly, the system revalidates your localized editor. The event broadcasts and both indicators turn red.